inc/views/partials/post_meta.php

```diff
@@ -298,7 +299,18 @@ public static function neve_get_author_meta( $post_id = null, $show_before = tru
 
 		$markup .= wp_kses_post( $link ) . '</span>';
 
-		return $markup;
+		/**
+		 * Filters the author post meta markup.
+		 *
+		 * @since 3.5.0
+		 *
+		 * @param int | null $post_id Post id.
+		 * @param $show_before bool Show the 'by' sting before the author name.
+		 * @param bool $markup The HTML markup used to display the post meta author.
+		 *
+		 * @return string
+		 */
+		return apply_filters( 'neve_filter_author_meta_markup', $post_id, $show_before, $markup );
```

Comment:
The first argument of the filter is the value, so `$markup` should be first then additional arguments. I am pretty sure this will return the `$post_id`. 

Maybe if possible also add a sanitization function before returning the output.

Reply:
Thank you, @preda-bogdan, for reviewing this. I've changed the order as you advised and added a sanitization function at the end.
Just a note here: the previous code before this was also working since the filter looked like this: `apply_filters( 'neve_filter_author_meta_markup', $post_id, $show_before, $markup );` and the function like this: `function neve_get_author_meta( $post_id, $show_before, $markup )`. As long as you respect the parameters order when using add_filter, everything should work well but in this case, it makes more sense to pass the markup as the first parameter.

Reply:
Yes but if no function is hooked on the filter the default would have been $post_id witch was not expected.

Reply:
@cristian-ungureanu I agree with @preda-bogdan the filter was wrongly been used, is not about a preference, that filter should return the markup, not the post_id in order to work properly.

Reply:
Ah, yes, you're right @preda-bogdan. Thanks for pointing this out.